### ISSUE TITLE
Issue when exporting composite

### ISSUE THREAD

**Issue by Mathieuduverney** [theskeletoncrew/treat-toolbox#39]
Hi everyone,

Another issue after #38
As said in this ticket, I'm only able to generate 100 unique NFT (see #38)

I tried to export these 100 for the candy machine, but I got this issue:

` download failed
>  ApiError: Not Found
>      at new ApiError (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/@google-cloud/common/build/src/util.js:73:15)
>      at Util.parseHttpRespMessage (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/@google-cloud/common/build/src/util.js:175:41)
>      at Util.handleResp (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/@google-cloud/common/build/src/util.js:149:76)
>      at Duplexify.<anonymous> (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/@google-cloud/storage/build/src/file.js:888:31)
>      at Duplexify.emit (events.js:315:20)
>      at PassThrough.emit (events.js:315:20)
>      at onResponse (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/retry-request/index.js:222:19)
>      at PassThrough.<anonymous> (/Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/retry-request/index.js:163:11)
>      at PassThrough.emit (events.js:327:22)
>      at /Users/bib-it/Documents/Perso/TreatBox_code/treat-toolbox/functions/node_modules/teeny-request/build/src/index.js:191:27
>      at processTicksAndRejections (internal/process/task_queues.js:97:5) {
>    code: 404,
>    errors: [],
>    response: PassThrough {
>      _readableState: ReadableState {
>        objectMode: false,
>        highWaterMark: 16384,
>        buffer: BufferList { head: null, tail: null, length: 0 },
>        length: 0,
>        pipes: null,
>        pipesCount: 0,
>        flowing: false,
>        ended: true,
>        endEmitted: true,
>        reading: false,
>        sync: false,
>        needReadable: false,
>        emittedReadable: false,
>        readableListening: false,
>        resumeScheduled: false,
>        emitClose: true,
>        autoDestroy: false,
>        destroyed: false,
>        defaultEncoding: 'utf8',
>        awaitDrainWriters: Set {},
>        multiAwaitDrain: true,
>        readingMore: false,
>        decoder: null,
>        encoding: null,
>        [Symbol(kPaused)]: false
>      },
>      readable: false,
>      _events: [Object: null prototype] {
>        prefinish: [Function: prefinish],
>        error: [Array],
>        close: [Array],
>        end: [Function: onend],
>        finish: [Function: onfinish]
>      },
>      _eventsCount: 5,
>      _maxListeners: undefined,
>      _writableState: WritableState {
>        objectMode: false,
>        highWaterMark: 16384,
>        finalCalled: false,
>        needDrain: false,
>        ending: true,
>        ended: true,
>        finished: true,
>        destroyed: false,
>        decodeStrings: true,
>        defaultEncoding: 'utf8',
>        length: 0,
>        writing: false,
>        corked: 0,
>        sync: false,
>        bufferProcessing: false,
>        onwrite: [Function: bound onwrite],
>        writecb: null,
>        writelen: 0,
>        afterWriteTickInfo: null,
>        bufferedRequest: null,
>        lastBufferedRequest: null,
>        pendingcb: 0,
>        prefinished: true,
>        errorEmitted: false,
>        emitClose: true,
>        autoDestroy: false,
>        bufferedRequestCount: 0,
>        corkedRequestsFree: [Object]
>      },
>      writable: false,
>      allowHalfOpen: true,
>      _transformState: {
>        afterTransform: [Function: bound afterTransform],
>        needTransform: false,
>        transforming: false,
>        writecb: null,
>        writechunk: null,
>        writeencoding: 'buffer'
>      },
>      statusCode: 404,
>      statusMessage: 'Not Found',
>      request: {
>        agent: false,
>        headers: [Object],
>        href: 'http://localhost:9199/b/the-elephant-house.appspot.com/o/bPnsEkYKnQGIscQBd7fb%2FFQ9L8rFa1ERVs1W3ENpf%2Fgenerated%2FraT2OhBxmw5hlLOCvqol%2F0.json?alt=media'
>      },
>      body: [Circular],
>      headers: {
>        'access-control-expose-headers': 'content-type,x-firebase-storage-version,x-goog-upload-url,x-goog-upload-status,x-goog-upload-command,x-gupload-uploadid,x-goog-upload-header-content-length,x-goog-upload-header-content-type,x-goog-upload-protocol,x-goog-upload-status,x-goog-upload-chunk-granularity,x-goog-upload-control-url',
>        connection: 'close',
>        'content-length': '9',
>        'content-type': 'text/plain; charset=utf-8',
>        date: 'Sat, 13 Nov 2021 09:17:30 GMT',
>        etag: 'W/"9-0gXL1ngzMqISxa6S1zx3F4wtLyg"',
>        vary: 'Origin',
>        'x-powered-by': 'Express'
>      },
>      toJSON: [Function: toJSON],
>      [Symbol(kCapture)]: false
>    },
>    bufferedData: <Buffer >
>  }`

Seems to be an issue with API

Do you have any idea?

**Comment by skeletoncrewrip** [comment]
Some users have had success switching to the OpenJDK for java when they faced this issue. Btw, you can try our #dev-skulls channel on discord too for troubleshooting help.

**Comment by Mathieuduverney** [comment]
Working now!
Didnt change anything on my end tho.

Closing the issue thanks!
